CMakeLists.txt

```diff
@@ -283,6 +287,10 @@ ELSE ()
 		FIND_PACKAGE(OpenSSL)
 	ENDIF ()
 
+	IF (NOT AMIGA AND USE_MBEDTLS)
```

Comment:
Not sure.  @sba1 made that change in commit c57c4af327979ad4189b108eaf751c912c8dea01.